### DISCUSSION TITLE
Changing Syntax for Commands: Help Needed

### DISCUSSION THREAD

**Discussion by mcarella** [evennia/evennia#3575] [Evennia Questions] [answered]
Hi everyone,

I've encountered an issue with the syntax used for commands in my project and I would like to ask your help.

Previously, the syntax for commands was formatted like this:

```
look 1-object
look 2-object
```

However, I recently noticed that the syntax has changed to:

```
look object-1
look object-2
```

It could be because the evennia version I'm using now differs from the one I've used in the past.

But anyway I have two main questions regarding this change:

**Rolling Back to the Old Syntax:**
How can I revert back to using the old syntax format (look 1-object, look 2-object, etc.)? Is there a configuration or setting that allows me to switch back?

**Changing the Separator:**
If I want to use a different separator instead of the hyphen, for example, using a dot (`look 1.object, look 2.object`), what steps do I need to take? Is there a specific file or script I need to modify to achieve this?

Any detailed instructions or pointers to relevant documentation would be greatly appreciated. Thanks in advance for your help!

**Comment by InspectorCaracal** [comment, marked as answer]
Hey there! I think this is exactly what you're looking for: https://github.com/evennia/evennia/discussions/3427

**Reply by mcarella** [reply]
That's exactly what I was looking for, super-thanks! :-)
